drivers/hmis/app/graphql/mutations/rename_service_type_type.rb

```diff
@@ -0,0 +1,25 @@
+#  Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+#  License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+#
+
+module Mutations
+  class RenameServiceTypeType < BaseMutation
```

Comment:
I think you can add the `graphql_name` property here to specify `RenameServiceType` , and rename the class and file name. Looks like Mutation has it: https://graphql-ruby.org/api-doc/1.8.7/GraphQL/Schema/Mutation.html

Reply:
Perfect!! Thank you